packages/platform/atoms/cal-provider/CalProvider.tsx

```diff
@@ -10,12 +10,26 @@ export type CalProviderProps = {
   clientId: string;
   accessToken: string;
   options: { refreshUrl?: string; apiUrl: string };
+  timezoneUpdatePreference?: "auto" | "none";
```

Comment:
I think instead of having "timezoneUpdatePreference" is clearer to have "updateTimezoneAutomatically: boolean" with default set to true - because "timezoneUpdatePreference" right now is optional, so if it would not be passed, is it the same as passing "none" and having optional argument with "none" option that disables its functionality IMO is confusing.

Reply:
True it's better to have updateTimezoneAutomatically prop with a boolean, makes it less confusing